doc/api/deprecations.md

```diff
@@ -2336,6 +2336,20 @@ Type: Runtime
 Setting the TLS ServerName to an IP address is not permitted by
 [RFC 6066][]. This will be ignored in a future version.
 
+<a id="DEP0124"></a>
```

Comment:
Should this be `DEP0XXX` here and below till landing?

Reply:
Yes it should

Reply:
Should I change the deprecations other than mine in this pull request? I think I would rather have that be a separate PR.